src/apify/_actor.py

```
@@ -50,6 +51,26 @@
 
 
 MainReturnType = TypeVar('MainReturnType')
+TFun = TypeVar('TFun', bound=Callable[..., Any])
+
+
+def _add_local_storage_error_hint(function: TFun) -> TFun:
+    """This decorator adds a local storage error hint in situation where storage was not found locally."""
+
+    @functools.wraps(function)
+    async def wrapper(
+        self: _ActorType, *, id: str | None = None, name: str | None = None, force_cloud: bool = False
+    ) -> Any:
+        try:
+            return await function(self=self, id=id, name=name, force_cloud=force_cloud)
+        except Exception as e:
```

Do we really want to catch all exceptions?